api/job-batches/params/job-batches-parameters.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseParameters } from "../../parameters";
+import { SmartlingException } from "../../exception";
+
+export class JobBatchesParameters extends BaseParameters {
+    setDescription(description: string): this {
+        if (description.length >= 2000) {
+            throw new SmartlingException("Job description should be less than 2000 characters.");
+        }
+
+        this.set("description", description);
+
+        return this;
+    }
+    setTargetLocaleIds(targetLocaleIds: string[]): this {
+        this.set("targetLocaleIds", targetLocaleIds);
+
+        return this;
+    }
+    setMode(mode: "CREATE_NEW" | "REUSE_EXISTING"): this {
+        this.set("mode", mode);
+
+        return this;
+    }
+    setSalt(salt: "ORDINAL" | "TIMESTAMP" | "RANDOM_ALPHANUMERIC"): this {
+        this.set("salt", salt);
+
+        return this;
+    }
+    setTimeZoneName(timeZoneName: string): this {
```

Comment:
Had a discussion with @dimitrystd , regarding validation for timeZoneName here.
I've looked up the source code of jobs-facade `com.smartling.jobs.facade.service.JobsService#getNow`, and can't think of any validation here. The code will fallback to GMT on unsupported values.

Reply:
Unfortunately, IANA dates and whatever Java uses are somewhat different sets, e. g., Java is ok with `GMT+8`, Intl.DateTimeFormat is not ok with that.

Reply:
Ok, let's leave it as it. I hope our API (java) returns a user-friendly error message.